projects/go-style-guide/src/app/features/ui-kit/components/table-docs/components/table-actions-docs/table-actions-docs.component.ts

```diff
@@ -27,6 +27,13 @@ export class TableActionsDocsComponent {
   </go-table>
   `;
 
+  buttonSizeExample: string = `
+  <ng-container go-table-actions>
+    <go-icon-button buttonIcon="filter" buttonSize="medium"></go-icon-button>
+    <go-icon-button buttonIcon="settings" buttonSize="medium"></go-icon-button>
+  </ng-container>
+  `;
+
```

Comment:
I think we can actually replace lines 18-19 in the above example with lines 32-33 here. `tableActions_html` is documenting how a table with actions should look, we should represent the `medium` icon size there and can probably remove this `buttonSizeExample`.

Thoughts?

Reply:
That works as well. I was just being more verbose, considering what if the user misses the 'buttonSize' feature in the larger example!